src/forms/services/form-submission.service.ts

```
@@ -0,0 +1,1184 @@
+
+    // Add conditions for each custom field filter
+    Object.entries(customFieldFilters).forEach(([fieldId, expectedValue]) => {
+      const fieldDef = fieldDefinitionMap.get(fieldId);
+      if (!fieldDef) {
+        throw new BadRequestException(`Invalid field ID: ${fieldId}`);
+      }
+
+      const fieldType = fieldDef.type;
+      switch (fieldType) {
+        case FieldType.NUMERIC:
+          const numValue = Number(expectedValue);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)